judgels-frontends/raphael/src/routes/uriel/contests/single/submissions/Bundle/modules/contestSubmissionActions.ts

```diff
@@ -23,14 +23,14 @@ export const contestSubmissionActions = {
     };
   },
 
-  getSummary: (contestJid: string, userJid?: string) => {
+  getSummary: (contestJid: string, userJid?: string, language?: string) => {
     return async (dispatch, getState, { contestSubmissionBundleAPI }) => {
       const token = selectToken(getState());
-      return contestSubmissionBundleAPI.getLatestContestantAnswersInContest(token, contestJid, userJid);
+      return contestSubmissionBundleAPI.getAnswerSummaryForContestant(token, contestJid, userJid, language);
     };
   },
 
-  getLatestSubmission: (contestJid: string, problemJid: string) => {
+  getLatestSubmissions: (contestJid: string, problemJid: string) => {
```

Comment:
Actually you have to change [ContestProblemPage.tsx at line 140](https://github.com/ia-toki/judgels/blob/3c8a6e84424d7a12e315558dba8b119d791b38cd/judgels-frontends/raphael/src/routes/uriel/contests/single/problems/single/ContestProblemPage/Bundle/ContestProblemPage.tsx#L140) too

Reply:
Oh, right. Can you fix it? :')